core/templates/dev/head/profile/profile.html

```diff
@@ -41,6 +41,13 @@ <h1 class="oppia-profile-username"><strong>{{PROFILE_USERNAME}}</strong></h1>
               </span>
             </p>
             <p>
+              <span ng-if="subjectInterests">
+                Subject Interests: <[subjectInterests]></span>
+              <span ng-if="!subjectInterests">
```

Comment:
OK, I think what you have is fine.

One nit: would you mind pulling the common "Subject Interests:" part out of the ng-ifs?